<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PiPeValve</class>
 <widget class="QWidget" name="PiPeValve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>55</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="valveIndexLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>24</y>
     <width>15</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>#</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="valvePushButton">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>0</y>
     <width>29</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>40</y>
     <width>16</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>槽风阀</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
